src/Tickets/Commerce/Gateways/PayPal/Assets.php

```diff
@@ -36,6 +36,7 @@ public function register() {
 			[],
 			'admin_enqueue_scripts',
 			[
+				'conditionals' => [ $this, 'should_enqueue_assets_payments_tab' ],
```

Comment:
Should we also modify the `should_enqueue_assets_payments_tab` to check if we're on the `paypal` tc-section? (now that we have one).

Probably this logic was implemented before we had "sections" on the Settings. And if we don't include that check, the assets will be loaded on other settings and in the payments tab in general.

Reply:
Correct, we should do that. Updated with the logic now.